product_docs/docs/migration_toolkit/55/installing/macos.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Installing on Mac OS X"
-navTitle: "On Mac OS X"
+title: "Installing on macOS"
+navTitle: "On macOS"
```

Comment:
We are no longer building macOS installers. This page and references to macOS should be removed.